### DISCUSSION TITLE
Does Wyze expose access to Floodlight "Auto" mode?

### DISCUSSION THREAD

**Discussion by rmk36** [SecKatie/ha-wyzeapi#531] [Q&A]
Thanks for providing this awesome tool!  It along with the [docker wyze bridge](https://github.com/mrlt8/docker-wyze-bridge/) are amazing tools for Wyze/HA users.

I've got the original [Wyze floodlight](https://www.wyze.com/products/wyze-cam-floodlight) and I normally trigger the floodlight to turn on automatically with PIR motion as shown here.

![image](https://github.com/SecKatie/ha-wyzeapi/assets/152319647/1423e5ae-6603-423e-8bda-01e576090b47)

It seems when I turn the floodlight on or off from within Home Assistant, the floodlight then ignores PIR motion activation unless I go in the Wyze app and reset the light to "Auto" mode.  

Interestingly, whenever I open to app to reset, the "Auto" selection is still visibly active, so I switch it to "On" and then back to "Auto."

Does the Wyze API allow us to put the floodlight into "Auto" mode (and as a bonus, do we have access to the flood light brightness level)?

Perhaps my understanding is incorrect, but as with most things Wyze, it seems a bit buggy and not always consistent in behavior.

**Comment by rmk36** [comment]
Total noob here, when it comes to API work.  I'm more of a user than a dev.  Pardon any stupid thoughts that follow...

I see in this [bit of code in the parent API repository](https://github.com/SecKatie/wyzeapy/blob/main/src/wyzeapy/services/camera_service.py) the floodlight property values is toggled between 1 and 2.

Is it possible a value of 0  (or 3 maybe?) is valid and is the "Auto" setting?

![image](https://github.com/SecKatie/ha-wyzeapi/assets/152319647/438ab1f1-b568-42d3-82c9-3147c37866ed)

**Comment by ryanwnzl** [comment]
Is there any way to control any floodlight features (Delay/Brightness/etc) from HA?

The docker is great for video and this app allows On/Off, but I would like to be able to control brightness &delay it like anything else, any suggestions?
